### ISSUE TITLE
tldr curl exception

### ISSUE THREAD

**Issue by zhukovra** [BrainMaestro/tldr-php#1]
`tldr curl` cause
```                                         
  [Symfony\Component\Console\Exception\InvalidArgumentException]  
  Incorrectly nested style tag found.                             
```

**Environment**:
* PHP 7.1.11-1+ubuntu16.04.1+deb.sury.org+1 (cli)
* Tldr v0.1.0                                                            

**Verbosed output**:
```
tldr -vvv curl

                                                                  
  [Symfony\Component\Console\Exception\InvalidArgumentException]  
  Incorrectly nested style tag found.                             
                                                                  

Exception trace:
 () at /home/rmn/.composer/vendor/symfony/console/Formatter/OutputFormatterStyleStack.php:75
 Symfony\Component\Console\Formatter\OutputFormatterStyleStack->pop() at /home/rmn/.composer/vendor/symfony/console/Formatter/OutputFormatter.php:165
 Symfony\Component\Console\Formatter\OutputFormatter->format() at /home/rmn/.composer/vendor/symfony/console/Output/Output.php:155
 Symfony\Component\Console\Output\Output->write() at /home/rmn/.composer/vendor/symfony/console/Output/Output.php:132
 Symfony\Component\Console\Output\Output->writeln() at /home/rmn/.composer/vendor/brainmaestro/tldr/src/Commands/Tldr.php:73
 BrainMaestro\Tldr\Commands\Tldr->execute() at /home/rmn/.composer/vendor/symfony/console/Command/Command.php:240
 Symfony\Component\Console\Command\Command->run() at /home/rmn/.composer/vendor/symfony/console/Application.php:858
 Symfony\Component\Console\Application->doRunCommand() at /home/rmn/.composer/vendor/symfony/console/Application.php:216
 Symfony\Component\Console\Application->doRun() at /home/rmn/.composer/vendor/symfony/console/Application.php:122
 Symfony\Component\Console\Application->run() at /home/rmn/.composer/vendor/brainmaestro/tldr/tldr:25

tldr [-p|--platform PLATFORM] [-c|--clear-cache] [-u|--update] [--] [<page>]
```

**Comment by BrainMaestro** [comment]
Hey @zhukovra. This is most likely because the format for the tldr pages changed.

**Comment by owenvoke** [comment]
It looks like this is caused by Line 16 of the `curl.md` file.

```
`curl -O -L -C - {{http://example.com/filename}}`
```

It doesn't seem to like the use of ` - ` in this bit of the `Page` class.
https://github.com/BrainMaestro/tldr-php/blob/86ca367fc4aa398fa7b6a3cd0c123da73cb53e93/src/Page.php#L86

Seems this can be fixed with `/^(- .+)/m`.

**Comment by owenvoke** [comment]
Feel free to check over that, but I've run some tests and it seems ok. 🤔

**Comment by zhukovra** [comment]
Fixed. Thanks @pxgamer
